beacon-chain/rpc/eth/beacon/handlers_test.go

```diff
@@ -515,7 +515,7 @@ func TestGetBlockAttestations(t *testing.T) {
 			Blocker:               mockBlockFetcher,
 		}
 
-		request := httptest.NewRequest(http.MethodGet, "http://foo.example/eth/v2/beacon/blocks/{block_id}/attestations", nil)
+		request := httptest.NewRequest(http.MethodGet, "http://foo.example/eth/v1/beacon/blocks/{block_id}/attestations", nil)
```

Comment:
I think it would be cleaner to organize tests like this:
```
t.Run("v1")
t.Run("v2")
    t.Run("pre-Electra")
    t.Run("post-Electra")
```
Blocks can be created at the top, before test runs.